SparseGrids/tsgCoreOneDimensional.cpp

```diff
@@ -67,13 +67,13 @@ template<bool iomode> void CustomTabulated::read(std::istream &is){
         std::string T;
         char dummy;
         is >> T;
-        if (!(T.compare("description:") == 0)){ throw std::invalid_argument("ERROR: wrong file format of custom tables on line 1"); }
+        if (!(T.compare("description:") == 0)){ throw std::runtime_error("ERROR: wrong file format of custom tables on line 1"); }
```

Comment:
Do not change these, someone may already be trying to capture those exceptions. Effectively, this changes the external API.

Catch two exceptions in the `tsgReadCustomTabulated()`, that's a better way to do it.

Reply:
Done.